pkg/storaged/utils.js

```diff
@@ -1024,6 +1034,12 @@
         }
     }
 
+    async function stop_swap(swaps) {
+        for (const s of swaps) {
+            await client.blocks_swap[s.block.path].Stop({});
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240116-090759-3ab49c70-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)